lib/core/themes/styles.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+import 'package:mentorship_e1_g3/core/extension/num_extension.dart';
+import 'package:mentorship_e1_g3/core/themes/app_pallete.dart';
+
+class AppStyles{
+    static TextStyle bold24(BuildContext context) {
```

Comment:
it's better if we name bold24 >>> font24WhiteBold or font24BoldWhite , what do u think ?